test/spec/codeTest2/codeTest2.js

```diff
@@ -0,0 +1,117 @@
+const { expect } = require("chai");
+const moment = require("moment");
+
+//Q1
+describe('WebDriver IO Code Test', () => {
+        // Question - 1: (50-points)
+        /**
+         * Testcase: Verify rewards form is empty and Conitnue button is disabled
+         * Steps:
+         * 1. Launch hotels.com
+         *      -Scroll to step 2
+         * 2. Click on Learn about Hotels.com Rewards
+         * 3. -> Verify Hotels Rewards opened in a new window
+         * 4. Click on Join Now
+         * 5. -> Verify Form is blank
+         * 6. -> Verify Continue button is NOT enabled
+         */
+        it('Verify rewards form is empty and Conitnue button is disabled', async () => {
+    
+            //1. Launch hotels.com
+            await browser.url('https://www.hotels.com/');
+            await browser.pause(2000);
+           
+            // 2. Click on Learn about Hotels.com Rewards
+             //    -Scroll to step 2
+            const rewardsSection = await $('a=Learn about Hotels.com Rewards');
+            await rewardsSection.scrollIntoView(false);
+            await rewardsSection.click();
+
+                // Wait for new window to open
+    await browser.waitUntil(async () => {
+        const handles = await browser.getWindowHandles();
+        return handles.length === 2;
+    });
+
+            //3. -> Verify Hotels Rewards opened in a new window
+                // Before verifying switch handle
+        //switching handles
+        const connectedHandle = await browser.getWindowHandle();
+        console.log(`connectedHandle -> ${connectedHandle}\n\n`);
+
+        const allHandles = await browser.getWindowHandles();
+        console.log(allHandles);
```

Comment:
You don't need to print everything for actual testing purpose but you can do it for your debugging purpose. So before pushing to repo you can delete all logs.